sources/packages/backend/libs/utilities/src/math-utils.ts

```diff
@@ -10,3 +10,28 @@ export function decimalRound(decimalValue: number, decimalPlaces = 2): number {
   const multiplier = Math.pow(10, decimalPlaces);
   return Math.round(decimalValue * multiplier) / multiplier;
 }
+
+/**
+ * Ensures that a number is rounded up to the decimal places
+ * and have the decimal part combined into the integer part.
+ * For instance:
+ * - 12345.78999 = 1234579 (79 is the decimal part).
+ * - 123.999 = 12400 (00 is the decimal part).
+ * - 123.9 = 12390 (90 is the decimal part).
+ * - 123 = 12300 (00 is the decimal part).
+ * @param decimalNumber decimal number to be rounded.
+ * @param decimalPlaces number of decimal places to be considered
+ * for rounding.
+ * @returns decimal number converted to a integer where the decimals
+ * part are rounded and combined into the integer part.
+ */
+export function combineDecimalPlaces(
```

Comment:
Not a show stopper, just a suggestion, same thing using the exponentiation operator (**).

export function combineDecimalPlaces(decimalNumber: number, decimalPlaces = 2): number {
  const multiplier = 10 ** decimalPlaces;
  return Math.round(decimalNumber * multiplier) / multiplier;
}